source/agora/node/Validator.d

```diff
@@ -91,6 +99,50 @@ public class Validator : FullNode, API
         // currently we are not saving preimage info,
         // we only have the commitment in the genesis block
         this.regenerateQuorums(Height(0));
+
+        if(config.validator.registry_address.length && config.validator.addresses_to_register.length)
```

Comment:
This needs to be validated when the config file is parsed, not when the validator is instantiated

Reply:
the reason why I validated it here is that I assumed having a correct name registry is not mandatory requirement for a Validator to run